### PR TITLE
build off release/0.8

### PR DESCRIPTION
**Pull request by mattip**
As requested, rebuilt the documentation for 0.8.0 off release/0.8 HEAD. Note I also needed to modify functional/functional.py to add a blank line before the `Args` [here](https://github.com/pytorch/audio/blob/v0.8.0-rc3/torchaudio/functional/functional.py#L1015) (from the rc3 tag), which I see is fixed on HEAD.

@brianjo

### CODE REVIEW

**Review by vincentqb — APPROVED** [review]
LGTM, I can see the changes from #1327.

File: 0.8.0/_modules/torchaudio/functional/functional.html [hunk 1]
```diff
@@ -1311,17 +1311,21 @@ <h1>Source code for torchaudio.functional.functional</h1><div class="highlight">
 <span class="p">)</span> <span class="o">-&gt;</span> <span class="n">Tensor</span><span class="p">:</span>
     <span class="sa">r</span><span class="sd">&quot;&quot;&quot;</span>
 <span class="sd">    Applies codecs as a form of augmentation</span>
+
 <span class="sd">    Args:</span>
 <span class="sd">        waveform (Tensor): Audio data. Must be 2 dimensional. See also ```channels_first```</span>
 <span class="sd">        sample_rate (int): Sample rate of the audio waveform</span>
 <span class="sd">        format (str): file format</span>
```
Comment (vincentqb) [inline 0.8.0/_modules/torchaudio/functional/functional.html:1318]: I do not see the change made [here](https://github.com/pytorch/audio/pull/1317/files#diff-f69839488ce70c39d8c3db8c87790ac963364faf8aefcb38ff5b026ee9dca8c7R1019) in #1317 ?

Comment (mthrok) [reply 0.8.0/_modules/torchaudio/functional/functional.html:1318]: this file is for showing the source code. the one you are looking for are `0.8.0/transforms.html` `0.8.0/functional.html`

**Review by mthrok — APPROVED** [review]
@mattip 

Thanks for this.

File: 0.8.0/_modules/torchaudio/functional/functional.html [hunk 2]
```diff
@@ -1311,17 +1311,21 @@ <h1>Source code for torchaudio.functional.functional</h1><div class="highlight">
 <span class="p">)</span> <span class="o">-&gt;</span> <span class="n">Tensor</span><span class="p">:</span>
     <span class="sa">r</span><span class="sd">&quot;&quot;&quot;</span>
 <span class="sd">    Applies codecs as a form of augmentation</span>
```
Comment (vincentqb) [inline 0.8.0/_modules/torchaudio/functional/functional.html:1313]: This also does not include the change from #1317.

File: 0.8.0/functional.html [hunk 3]
```diff
@@ -480,112 +471,107 @@ <h3>biquad<a class="headerlink" href="#biquad" title="Permalink to this headline
...
-<dd><p>Apply ISO 908 CD de-emphasis (shelving) IIR filter.  Similar to SoX implementation.</p>
+<dt id="torchaudio.functional.apply_codec">
+<code class="sig-prename descclassname">torchaudio.functional.</code><code class="sig-name descname">apply_codec</code><span class="sig-paren">(</span><em class="sig-param">waveform: torch.Tensor</em>, <em class="sig-param">sample_rate: int</em>, <em class="sig-param">format: str</em>, <em class="sig-param">channels_first: bool = True</em>, <em class="sig-param">compression: Optional[float] = None</em>, <em class="sig-param">encoding: Optional[str] = None</em>, <em class="sig-param">bits_per_sample: Optional[int] = None</em><span class="sig-paren">)</span> &#x2192; torch.Tensor<a class="reference internal" href="_modules/torchaudio/functional/functional.html#apply_codec"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#torchaudio.functional.apply_codec" title="Permalink to this definition">¶</a></dt>
+<dd><p>Applies codecs as a form of augmentation</p>
```
Comment (vincentqb) [inline 0.8.0/functional.html:531]: This also does not show the changes in #1317.

File: 0.8.0/_modules/torchaudio/functional/functional.html (hunk 1 above)
Comment (vincentqb) [reply 0.8.0/_modules/torchaudio/functional/functional.html:1318]: Looking at 0.8.0/functional.html shows the outdated version too, see [comment](https://github.com/pytorch/audio/pull/1354#discussion_r587797575) below.

Comment (mthrok) [reply 0.8.0/_modules/torchaudio/functional/functional.html:1318]: oh you are right.

**Pull request merged**